graphql_api/tests/test_components_legacy.py

```diff
@@ -0,0 +1,1389 @@
+from unittest.mock import PropertyMock, patch
```

Comment:
This file will get cleaned up with #2290

Reply:
It is a copy of the existing `graphql_api/tests/test_components.py` original file